recipes/libcamera/meta.yaml

```
@@ -0,0 +1,64 @@
+{% set name = "libcamera" %}
+{% set version = "0.4.0" %}
+
+package:
+  name: {{ name|lower }}
```

```suggestion
  name: libcamera
```

extra variables don't bring too much value tbh..